src/Value/Color.php

```diff
@@ -30,104 +30,125 @@ public function __construct(array $aColor, $iLineNo = 0)
      */
     public static function parse(ParserState $oParserState, bool $bIgnoreCase = false): CSSFunction
     {
-        $aColor = [];
         if ($oParserState->comes('#')) {
-            $oParserState->consume('#');
-            $sValue = $oParserState->parseIdentifier(false);
-            if ($oParserState->strlen($sValue) === 3) {
-                $sValue = $sValue[0] . $sValue[0] . $sValue[1] . $sValue[1] . $sValue[2] . $sValue[2];
-            } elseif ($oParserState->strlen($sValue) === 4) {
-                $sValue = $sValue[0] . $sValue[0] . $sValue[1] . $sValue[1] . $sValue[2] . $sValue[2] . $sValue[3]
-                    . $sValue[3];
-            }
+            return self::parseHexColor($oParserState);
+        } else {
+            return self::parseColorFunction($oParserState);
+        }
```

Comment:
This should either be a guard clause (with the code in the `else' branch then without the `else` keyword, or something along those lines (which is what I'd prefer):

```php
return $oParserState->comes('#') ? self::parseHexColor($oParserState) : self::parseColorFunction($oParserState);
```

Reply:
Yes, I had wondered about that given that I actually reduced the `return` statement in `parseColorFunction` using the ternary conditional operator.  Changed.